### PR TITLE
Updated README to include docker run example and advanced instructions 

### PR DESCRIPTION
**Pull request by rursache**
Updated README to include docker run example and advanced instructions on how to access the `user` folder and easly install plugins/custom pages or a public homepage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,36 @@ To run YOURLS service with customized config
     $ vim env.yourls
     $ docker compose up [--build] [-d]
 
+## Docker run
```
Comment (guessi) [inline README.md:52]: Could you please adjust the section name here and lower it down to `###` but not `##`, and leave a blank line after section title, thanks!

```
### Run with docker compose

...

### Run with docker

...

```

File: README.md [hunk 2]
```diff
@@ -49,6 +49,36 @@ To run YOURLS service with customized config
...
+  -e MYSQL_PASSWORD=yourls \
+  -e MYSQL_ROOT_PASSWORD=yourls-mysql \
+  -v /home/$USER/.yourls/mysql:/var/lib/mysql \
+  mysql:5.7
+
+docker run -d \
+  --name yourls \
+  --restart unless-stopped \
+  -p 80:80 \
+  -e TZ=Europe/Bucharest \
```
Comment (guessi) [inline README.md:71]: Prefer `UTC/Etc` as sample.

File: README.md [hunk 3]
```diff
@@ -49,6 +49,36 @@ To run YOURLS service with customized config
...
+```sh
+docker run -d \
+  --name yourls-mysql \
+  --restart always \
+  -p 3306:3306 \
+  -e TZ=Europe/Bucharest \
+  -e MYSQL_DATABASE=yourls \
+  -e MYSQL_USER=yourls \
+  -e MYSQL_PASSWORD=yourls \
+  -e MYSQL_ROOT_PASSWORD=yourls-mysql \
```
Comment (guessi) [inline README.md:63]: Prefer aligned with the sample for compose

```
--env-file env.mysql
```

File: README.md [hunk 4]
```diff
@@ -49,6 +49,36 @@ To run YOURLS service with customized config
...
+docker run -d \
+  --name yourls \
+  --restart unless-stopped \
+  -p 80:80 \
+  -e TZ=Europe/Bucharest \
+  -e YOURLS_SITE="https://example.com" \
+  -e YOURLS_DB_HOST=IP:3306 \ # mysql host:port
+  -e YOURLS_DB_NAME=yourls \ # mysql MYSQL_DATABASE
+  -e YOURLS_DB_USER=yourls \ # mysql MYSQL_USER
+  -e YOURLS_DB_PASS=yourls \ # mysql MYSQL_PASSWORD
```
Comment (guessi) [inline README.md:76]: same here

```
--env-file env.yourls
```

**Review by guessi — CHANGES_REQUESTED** [review]
@rursache Just left some comments, aim to keep doc aligned. Could you please check and make some updates to the PR, thanks!

**Comment by rursache** [comment]
@guessi 
i just wanted to add some examples extracted from my config. 
docker run and compose syntax is different and the way i put the example there is a very easy copy paste instead of also grabbing env files. you can see examples of this on linuxserver images.
i prefer it that way, if you don't, please apply the changes you want.

regarding the markdown format and timezone, sure, whatever works for you

**Pull request closed without merging**